crates/env/src/backend.rs

```diff
@@ -115,8 +115,15 @@ pub trait EnvBackend {
     where
         R: scale::Encode;
 
-    /// Prints the given contents to the console log.
-    fn println(&mut self, content: &str);
+    /// Emit a custom debug message.
+    ///
+    /// The message is appended to the debug buffer which is then supplied to the calling RPC
+    /// client. This buffer is also printed as a debug message to the node console if the
+    /// `debug` log level is enabled for the `runtime::contracts` target.
+    ///
+    /// This is a no-op if debug message recording is disabled which is always the case
+    /// when the code is executing on-chain.
+    fn debug_message(&mut self, content: &str);
```

Comment:
I am not sure but maybe we should rename it to `debug_println` for parity with SEAL API?